clients/input_copy.go

```diff
@@ -34,7 +34,7 @@ type InputCopy struct {
 
 // CopyInputToS3 copies the input video to our S3 transfer bucket and probes the file.
 func (s *InputCopy) CopyInputToS3(requestID string, inputFile *url.URL) (inputVideoProbe video.InputVideo, signedURL string, osTransferURL *url.URL, err error) {
```

Comment:
We copy here only the manifest file. Don't we need to copy segment files as well?

Reply:
I think that for V1, we can assume Studio / Recording is the only user of this API if that makes things simpler. I think that then we shouldn't have to worry about copying anything around since we know it'll be on Google Storage

Reply:
* We're not copying anything here if it's a directupload or hls-input - we can source the manifest directly and parse it and send it to the transcode session. 
* Copying segments around should be out of scope for recordings v2 since it is required for object stores other than GCS. So I submitted VID-144 as a backlog to address this once we ship the first version. Also see my comments below for additional context.

Reply:
The code was updated to handle generic HLS input in catalyst.